src/reynir_correct/tools/diffchecker.py

```diff
@@ -60,17 +63,22 @@ def main():
         raise ValueError
     itering = gen(inputfile)
     for sent in itering:
-        print("=================================")
+        if not sent.strip():
+            continue
         if sent.startswith("#"):
             # Comment string, want to show it with the examples
-            print(sent.strip())
-            sent = next(itering)
+            if not options["ignore_comments"]:
+                print(sent.strip())
+            # sent = next(itering)
+            continue
         options["infile"] = sent
         x = check_errors(**options)
         # Hér væri auðvelt að bæta við einhverjum samanburði við gull, skila bara T/F hvort sé eins
         # Þyrfti þá að zippa  saman setningar úr testskjali og gullskjali til að nota í forlykkju
         print(sent.strip())
-        print(x)
+        if x:
```

Comment:
Translate comment above to English? Or delete it?

Reply:
Done